platformio.ini

```diff
@@ -820,11 +788,35 @@ monitor_speed = 500000
 #
 # MKS RUMBA32 (adds TMC2208/2209 UART interface and AUX-1)
 #
-[env:rumba32_mks]
+[env:rumba32_mks_old]
 platform    = ${common_rumba32.platform}
 extends     = common_rumba32
 build_flags = ${common_rumba32.build_flags} -DUSBD_VID=0x8000
 
+#
+# RUMBA32
+#
+[env:rumba32_f446ve]
+platform      = ${common_stm32.platform}
+extends       = common_stm32
+board         = rumba32_f446ve
+build_flags   = ${common_stm32.build_flags}
+  -DSTM32F4xx -DARDUINO_RUMBA32_F446VE -DARDUINO_ARCH_STM32 "-DBOARD_NAME=\"RUMBA32_F446VE\""
+  -DSTM32F446xx "-DUSB_PRODUCT=\"RUMBA32_F446VE\""
+  -DHAL_PCD_MODULE_ENABLED
+  -DDISABLE_GENERIC_SERIALUSB
+  -DHAL_UART_MODULE_ENABLED
+  -Os
+monitor_speed = 500000
+upload_protocol = dfu
+
+#
+# MKS RUMBA32 (adds TMC2208/2209 UART interface and AUX-1)
+#
+[env:rumba32_mks]
+platform      = ${common_stm32.platform}
+extends       = env:rumba32_f446ve
```

Comment:
I don't know if it matters, but the `-DUSBD_VID=0x8000` flag is lost by this change. Does the MKS version of the board require this to be overridden here?

Reply:
Looks like you can override with a gcc -U so it is added below.

Reply:
@thinkyhead I had intentionally removed that. I did not see any reason why that one MKS board should have a different vendor ID than every other STM32 board in Marlin. It seemed pretty undesirable to just use a made up vendor ID.

Reply:
Honestly, I don't see why the separate rumba_mks environment even exists. It didn't seem different enough from the other rumba environment to be needed.

Reply:
It doesn't matter too much what we do with the USB stuff right now, because it all ends up changing in version 1.9 of the framework anyway.